.github/workflows/docker.yml

```diff
@@ -17,28 +17,128 @@ on:
     types: [opened, synchronize, reopened, ready_for_review]
 
 env:
-  FOREST_TEST_IMAGE: forest-image
-  FOREST_TEST_ARCHIVE: forest-image.tar
+  CI: 1
+  CARGO_INCREMENTAL: 0
+  CACHE_TIMEOUT_MINUTES: 5
+  SCRIPT_TIMEOUT_MINUTES: 30
+  AWS_ACCESS_KEY_ID: '${{ secrets.AWS_ACCESS_KEY_ID }}'
+  AWS_SECRET_ACCESS_KEY: '${{ secrets.AWS_SECRET_ACCESS_KEY }}'
+  RUSTC_WRAPPER: sccache
+  CC: sccache clang
+  CXX: sccache clang++
 
 jobs:
-  build-and-push-docker-image:
+  build-ubuntu-2204-amd64:
+    name: Build forest binaries on Ubuntu-22.04-amd64
+    runs-on: ubuntu-22.04
     # Run the job only if the PR is not a draft.
     # This is done to limit the runner cost.
     if: github.event.pull_request.draft == false
+    steps:
+      - name: Configure SCCache variables
+        run: |
+          # External PRs do not have access to 'vars' or 'secrets'.
+          if [[ "${{secrets.AWS_ACCESS_KEY_ID}}" != "" ]]; then
+            echo "SCCACHE_ENDPOINT=${{ vars.SCCACHE_ENDPOINT}}" >> $GITHUB_ENV
+            echo "SCCACHE_BUCKET=${{ vars.SCCACHE_BUCKET}}" >> $GITHUB_ENV
+            echo "SCCACHE_REGION=${{ vars.SCCACHE_REGION}}" >> $GITHUB_ENV
+          fi
+      - run: lscpu
```

Comment:
any specific reason to do this?

Reply:
It does not hurt to know more info about the buildjet host and it can be used for troubleshooting build errors. e.g. https://github.com/ChainSafe/forest/pull/2859